<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Manikandan" failures="1" tests="3" name="user-john" time="24.887" errors="0" timestamp="2025-07-12T21:35:09 IST">
  <testcase classname="com.vinsguru.tests.vendorportal.VendorPortalTest" name="loginTest" time="4.597"/>
  <testcase classname="com.vinsguru.tests.vendorportal.VendorPortalTest" name="dashboardTest" time="4.829">
    <failure type="java.lang.AssertionError" message="expected [0] but found [99]">
      <![CDATA[java.lang.AssertionError: expected [0] but found [99]
at com.vinsguru.tests.vendorportal.VendorPortalTest.dashboardTest(VendorPortalTest.java:50)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- dashboardTest -->
  <testcase classname="com.vinsguru.tests.vendorportal.VendorPortalTest" name="logoutTest" time="0.0">
    <skipped/>
  </testcase> <!-- logoutTest -->
</testsuite> <!-- user-john -->
